app/models/rebate_form.rb

```diff
@@ -2,14 +2,38 @@
 
 class RebateForm < ApplicationRecord
   has_many :signatures, dependent: :destroy
-  belongs_to :property, required: false
+  belongs_to :property, required: true
```

Comment:
This is new! Is the property doesn't exist in our data set, then you can't apply for a rebate for it.